### PR TITLE
MOTECH-2838 OpenMRS added Get CohortQuery Report Task action

### PR DESCRIPTION
**Pull request by filgomul**
I will add unit tests in separate PR

### CODE REVIEW

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by motech-gerrit** [comment]
Can one of the admins verify this patch?

**Comment by kmadej** [comment]
please resolve conflicts

**Comment by kmadej** [comment]
test this please

File: openmrs/src/main/java/org/motechproject/openmrs/helper/EventHelper.java [hunk 1]
```diff
@@ -195,4 +196,14 @@ private EventHelper() {
 
         return parameters;
     }
+
+    public static Map<String, Object> cohortMemberParameters(String cohortQueryUuid, CohortQueryReportMember member) {
```
Comment (kmadej) [inline openmrs/src/main/java/org/motechproject/openmrs/helper/EventHelper.java:201]: I think that we should send this event in task action logic

Additionally used string constants should be added to task package into appropriate class

**Comment by martokarski** [comment]
test this please

**Pull request merged**